src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -430,6 +431,19 @@ public List<Path> run() {
             }
         }
 
+        try {
+            String maskingAlgorithmDefault = settings.get(ConfigConstants.SECURITY_MASKED_FIELDS_ALGORITHM_DEFAULT);
+            if (StringUtils.isNotEmpty(maskingAlgorithmDefault)) {
+                MessageDigest.getInstance(maskingAlgorithmDefault);
+            }
+        } catch (Exception ex) {
+            throw new OpenSearchSecurityException(
```

Comment:
Can we add a test case for this scenario and ensure the node fails to boot up as expected?

What happens if a user explicitly puts `blake2b` in for this setting? Will the `MessageDigest.getInstance(...)` call fail since its not in this [list](https://docs.oracle.com/javase/8/docs/technotes/guides/security/StandardNames.html#MessageDigest)?

Reply:
That is the purpose of FieldMaskedTest.testMaskedGetClusterDefaultDoesNotExist()